src/QGISStyleParser.ts

```diff
@@ -24,6 +24,8 @@ import {
   Builder
 } from 'xml2js';
 
+const OUTPUT_VERSION = '3.28.0-Firenze';
```

Comment:
It might make sense to add this as a constructor parameter and add a switch to check if props or Option should be used.

This way it would still be possible to read and write older qmls and keep the input and output similiar.

What do you think?

Reply:
Done.
Also I've updated the commit messages in line with commitlint, so I've had to force-push